staging/src/github.com/kubeedge/beehive/pkg/core/context/context_factory.go

```diff
@@ -20,8 +20,12 @@ var (
 	once    sync.Once
 )
 
-// GetContext gets global context instance
-func GetContext(contextType string) *Context {
+func init()  {
```

Comment:
Do we need this `init` here? We have already call the `InitContext` in `StartModule` directly.

Reply:
ok